tools/plugins/com.liferay.ide.upgrade.plan.core/src/com/liferay/ide/upgrade/plan/core/UpgradePlanCorePlugin.java

```diff
@@ -73,6 +150,50 @@ public void stop(BundleContext context) throws Exception {
 		super.stop(context);
 	}
 
+	private void _initOfflineOutline() throws Exception {
```

Comment:
Lets not use a preference service for storing this, just a static initializer should be fine.